Modules/Loadable/Volumes/SubjectHierarchyPlugins/qSlicerSubjectHierarchyVolumesPlugin.cxx

```diff
@@ -484,13 +484,13 @@ QString qSlicerSubjectHierarchyVolumesPlugin::tooltip(vtkIdType itemID)const
     imageData->GetDimensions(dimensions);
     double spacing[3] = {0.0,0.0,0.0};
     volumeNode->GetSpacing(spacing);
-    tooltipString.append( QString(" (Dimensions: %1x%2x%3  Spacing: %4mm x %5mm x %6mm)")
+    tooltipString.append( QString(" (" + tr("Dimensions:") + " %1x%2x%3  " + tr("Spacing:") + " %4mm x %5mm x %6mm)")
       .arg(dimensions[0]).arg(dimensions[1]).arg(dimensions[2])
       .arg(spacing[0],0,'g',3).arg(spacing[1],0,'g',3).arg(spacing[2],0,'g',3) );
     }
   else
     {
-    tooltipString.append(" !Invalid volume");
+    tooltipString.append(" !" + tr("Invalid volume"));
```

Comment:
I don't think adding the `!` to the beginning is necessary, and `Invalid` is not specific enough. Let's do this instead:
```suggestion
    tooltipString.append(tr(" Empty volume!"));
```